examples/angular_bazel_architect/yarn.lock

```diff
@@ -10227,10 +10227,10 @@ webdriver-js-extender@2.1.0:
     "@types/selenium-webdriver" "^3.0.0"
     selenium-webdriver "^3.0.1"
 
-webdriver-manager@^12.1.7:
-  version "12.1.7"
-  resolved "https://registry.yarnpkg.com/webdriver-manager/-/webdriver-manager-12.1.7.tgz#ed4eaee8f906b33c146e869b55e850553a1b1162"
-  integrity sha512-XINj6b8CYuUYC93SG3xPkxlyUc3IJbD6Vvo75CVGuG9uzsefDzWQrhz0Lq8vbPxtb4d63CZdYophF8k8Or/YiA==
+webdriver-manager@12.1.8, webdriver-manager@^12.1.7:
```

Comment:
Should this not be `^12.1.8` like in the `resolutions`? If you run `yarn` again does it change it at all?

Reply:
Good catch. Fixed.